.github/workflows/_build.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         # It is recommended to pin a Runner version specifically:
         # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: ['3.9', '3.10']
+        python: ['3.10', '3.11']
```

Comment:
What's the point of supporting two Python versions only? Wouldn't it make sense to use packages like [cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/) to manage the Python version and OS?

Reply:
I’m not familiar with that package, but it looks interesting. Let’s move your suggestion to a separate issue?

Reply:
> I’m not familiar with that package, but it looks interesting. Let’s move your suggestion to a separate issue?

Sounds good, thanks.

Reply:
Issue https://github.com/jenstroeger/python-package-template/issues/451.